File: CSharp/modelo/ContaPoupanca.cs
using QualquerBank.interfaces;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QualquerBank.modelo
{
    public class ContaPoupanca : Conta, IRendimento
    {
        public ContaPoupanca(int numero, int agencia) : base(numero, agencia)
        {
        }

        
         public void Atualizar(double taxa)
        {
            Saldo += (Saldo * taxa);
        }

        public override void ExibeConta()
        {
            Console.WriteLine("Conta Poupanca");
            Console.WriteLine("Numero: " + Numero);
            Console.WriteLine("Agencia: " + Agencia);
            Console.WriteLine("Saldo: " + Saldo);
        }
    }
}

File: Java/contas/ContaPoupanca.java
package contas;

import interfaces.IRendimento;

public class ContaPoupanca extends Conta implements IRendimento{

	public ContaPoupanca(int numero, int agencia) {
		super(numero, agencia);
	}

	@Override
	public void exibeConta() {
		System.out.println("Conta Poupanca");
		System.out.println("Numero: " + getNumero());
		System.out.println("Agencia: " + getAgencia());
		System.out.println("Saldo: " + getSaldo());		
	}

	@Override
	public void atualizar(double taxa) {
		saldo += (saldo * taxa);		
	}
}